templates/default/contentlayer.config.js

```diff
@@ -17,6 +17,7 @@ const sharedFields = {
   description: { type: 'string' },
   image: { type: 'string' },
   layout: { type: 'string', default: 'docs' },
+  editLink: { type: 'boolean', default: siteConfig.editLinkShow},
```

Comment:
Yes, this is what I meant originally, but given the suggestion in my comment to `Layout`'s component logic, we won't need this default value here. So we would do:
```suggestion
  editLink: { type: 'boolean' },
```